### PR TITLE
Fix void zone click and column resize issues

### PR DESCRIPTION
**Pull request by ibdafna**

### CODE REVIEW

File: tests/test_datagrid.py [hunk 1]
```diff
@@ -141,6 +141,8 @@ def test_data_object_generation(dataframe: pd.DataFrame) -> None:
         ],
     }
 
+    print(data_object)
```
Comment (martinRenou) [inline tests/test_datagrid.py:144]: Maybe a leftover?

**Review by martinRenou — CHANGES_REQUESTED** [review]

**Comment by martinRenou** [comment]
Looks good! We should probably upstream this `isResizing` getter

**Review by gaborbernat — APPROVED** [review]

**Review by gaborbernat — APPROVED** [review]

**Comment by ibdafna** [comment]
Test failures in 3.7/3.6 due to pandas versions. Will update separately

**Pull request merged**